### DISCUSSION TITLE
(Newbie) Converting DV file only to HDR compatible stream

### DISCUSSION THREAD

**Discussion by AlbyGNinja** [quietvoid/dovi_tool#250] [General]
Hello, I'm trying to convert a bunch of files in dolby vision only that my TV won't play and watching them from my pc i see them with the classic purple green tone.
As I understand I extract the hevc only from the mkv file and a no-video.mkv file to be muxed with the converted stream, doing like so:

```
## extract the audio
mkvmerge -o output_no_video.mkv --no-video original_dv.mkv

## extract the hevc video
mkvextract tracks original_dv.mkv 0:video_output.hevc

## convert the hevc stream to hdr compatible format
dovi_tool convert video_output.hevc -o output_converted.hevc --discard

## mux the no-video file with the converted hevc into definitive file:
mkvmerge -o output_converted.mkv 
```

Am I missing something? 
The final video is still with the purple green tone and not working on my TV, link nothing :-(

**Comment by quietvoid** [comment]
`dovi_tool` doesn't modify the video itself, just the metadata.
You should just try using [mpv](https://mpv.io) with `--vo=gpu-next` for Dolby Vision playback.

Otherwise there are ways to convert to HDR10 but it requires transcoding and it's something you can google too.

**Reply by AlbyGNinja** [reply]
Do you think injecting the new metadata should be enough to playback correctly?
At the moment the mediainfo of the file says:
`HDR format: Dolby Vision, Version 1.0, dvhe.05.06, BL+RPU`

**Reply by quietvoid** [reply]
No, the video is not watchable without both the metadata and a player that can convert it.

**Reply by AlbyGNinja** [reply]
This means that even now I succesfully converted DV P5 to P8 stream I cannot edit anything in order to playback the video from my tv right?
This is the new mediainfo string:
`HDR format: Dolby Vision, Version 1.0, dvhe.08.06, BL+RPU, SDR compatible`

(I clarify my use case, I playback files from plex to a Samsung smart tv which do not support DV but HDR)

**Reply by quietvoid** [reply]
All you have done is editing the metadata to profile 8, and now the video and metadata are incompatible.

If you need a file that plays on your TV, you still need to transcode it into HDR10.

**Reply by AlbyGNinja** [reply]
Which means use something like Davinci Resolve to export it with HDR10 right?

**Reply by quietvoid** [reply]
I don't know, never used Resolve for that.
I know ffmpeg with libplacebo supports the conversion.
